integrations/dav-server-fs-opendal/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+mod webdav_dir_entry;
+mod webdav_file;
+mod webdav_metadata;
+pub mod webdavfs;
```

Comment:
How about expose an `OpendalFs` like [`LocalFs`](https://docs.rs/dav-server/latest/dav_server/localfs/index.html)?

Reply:
> I think `OpendalFs` can't embody the WebDAV protocol

Hi, I didn't get this. I'm talking about expose `OpendalFs` only instead a public module `webdavfs`.

Users will use `dav_server_fs_opendal::OpendalFs` instead of `dav_server_fs_opendal::webdavfs::Xxx`